WhereAmIAgain/DTRDisplay.cs

```diff
@@ -81,28 +81,38 @@ private void OnFrameworkUpdate(Framework framework)
 
     public void UpdateDtrText()
     {
+        var preTextEnd = Config.FormatString.IndexOf("{");
+        var postTextStart = Config.FormatString.LastIndexOf("}") + 1;
+        var formattedString = Config.FormatString;
+        
         try
         {
-            var formattedString = Config.FormatString.Format(
+            formattedString = formattedString[preTextEnd..postTextStart].Format(
                 currentContinent?.Name.RawString ?? string.Empty,
                 currentTerritory?.Name.RawString ?? string.Empty,
                 currentRegion?.Name.RawString ?? string.Empty,
                 currentSubArea?.Name.RawString ?? string.Empty);
 
-            var filteredString = Regex.Replace(formattedString, "^[^\\p{L}\\p{N}]*|[^\\p{L}\\p{N}]*$", "");
+            // Remove separators from between blank values
+            formattedString = Regex.Replace(formattedString, "^[^\\p{L}\\p{N}]*|[^\\p{L}\\p{N}]*$", "");
 
             if (Config.ShowInstanceNumber)
             {
-                filteredString += GetCharacterForInstanceNumber(GetInstanceNumber());
+                formattedString += GetCharacterForInstanceNumber(GetInstanceNumber());
             }
-            
-            dtrEntry.Text = new SeStringBuilder().AddText(filteredString).BuiltString;
-            locationChanged = false;
+
+            formattedString = Config.FormatString[..preTextEnd] + formattedString + Config.FormatString[..postTextStart];

```

Comment:
Just double checking, in Kami's example the split was at the end for post text start, was that intentional change?

Reply:
Oh you literally just fixed this ldkfamsdfl ignore me <3